tests/unit/lib/models/ActionQueueEntry.spec.js

```diff
@@ -43,6 +43,25 @@ describe('ActionQueueEntry', () => {
             entry.getAttribute('details'), { roundTrip: true });
     });
 
+    it('should set an error when retrieving an unset attribute and ' +
+    '"required" option is set', () => {
+        const entry = ActionQueueEntry.create('sendPigeon')
+              .setAttribute('toCountry', 'Finland');
+        assert.strictEqual(
+            entry.getAttribute('cost', { required: true }), undefined);
+        assert(entry.getError());
+        assert.strictEqual(entry.getError().code, 400);
+        assert.strictEqual(entry.getError().message, 'MissingParameter');
+
+        // nested attributes should work too
```

Comment:
Seems like this should be a separate test since it is not checking for an error when a required attribute is unset.

Reply:
Ah, interesting, it looks this is just an erroneous cut&paste from the above test, will remove!